Source/MSBuildTools.Unity/Packages/com.microsoft.msbuildforunity/Editor/ProjectGenerator/MSBuildTemplates/UnixNetSupport.props

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (c) Microsoft Corporation.  All Rights Reserved.  See License.txt in the project root for license information.
+https://github.com/dotnet/fsharp/blob/master/fcs/netfx.props (license is MIT)
+ -->
+<Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
```

Comment:
>PropertyGroup [](start = 3, length = 13)

As a quick reduction of a lot of these conditions you could split this to multiple PropertyGroups each with a condition.

<PropertyGroup Condition="'$(OS)' == 'Unix'">
    <BaseFrameworkPathOverrideForMono ...
    <BaseFrameworkPathOverrideForMono ...
    <BaseFrameworkPathOverrideForMono ...
</PropertyGroup>
<PropertyGroup Condition="'$(BaseFrameworkPathOverrideForMono)' != ''>
    <FrameworkPathOverride ...
    ...
</PropertyGroup>

If you wanted to get fancy with duplication removal you could also refactor these to be ItemGroups with some batch processing on them, but that could also just happen if this list starts to grow again.

Reply:
So this file was taken almost verabitim from https://github.com/dotnet/fsharp/blob/master/fcs/netfx.props, and I don't wish to further change it apart from adding the link.